doc/src/hugo/content/guides/authors/design/patterns/command-event/same-event.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "1 - Same Event"
+description: "Same Event For Each Command"
+date: 2022-02-25T10:50:32-07:00
+draft: "false"
+weight: 10
+---
+
+# Same Event
+
+## Q: Should separate commands on the same entity all yield the same event, containing the exact changed entity?
+
+### Example:
+
+event UserModified {
+    id: MemberId,
+    info: Info,
+    members: MemberId*,
+    contacts: Contacts,
+}
+
+## Suggestions
+
+This pattern is an anti-pattern most of the time because it results in lack of transparency. In an event driven system
+it is important to be able to easily discover the corresponding event for a command or vice-versa.
```

Comment:
The above isn't really a suggestion.  You should suggest that each command have an event that indicates what changed. 

It isn't clear to me how this is a lack of transparency.  I think it is more a lack of distinguishing one event from another because structurally they are all the same.